docker-compose.yml

```
@@ -33,6 +33,7 @@ services:
       GOVDELIVERY_TOKEN: 'abc123'
       GOVDELIVERY_STAGING_SERVICE: 'True'
       DEPLOYMENT_ENV: 'vagov-dev'
+      CI: "true"
```

I think single quotes are preferred here
```suggestion
      CI: 'true'
```

---

changed